src/bundle/Resources/public/js/scripts/helpers/middle.ellipsis.js

```diff
@@ -4,12 +4,14 @@
             parseAll(entry.target);
         });
     });
-    const parseAll = (baseElement = doc) => {
+    const parse = (baseElement = doc) => {
         if (!baseElement) {
+            console.warn('No baseElement provided');
```

Comment:
Printing this warning for `null` but not for `undefined` may be confusing. Maybe we can create more/less specific message? Or maybe create a more specific check?